karma.conf.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./config/karma.config.js');
```

Comment:
The only reason I can think of keeping it would be to make the code base easier to understand. That said, I vote for a cleaner code base and I agree with @ppitonak removing this file and renaming the other config makes sense to me.